packages.json

```
@@ -23540,5 +23540,19 @@
     "description": "Composition pattern with event handling library in Nim",
     "license": "MIT",
     "web": "https://github.com/DavidMeagher1/composition"
+  },
+  {
+    "name": "oolib",
+    "url": "https://github.com/Glasses-Neo/OOlib",
+    "method": "git",
+    "tags": [
+      "nim",
+      "oop",
+      "metaprogramming",
+      "nimble"
```

```suggestion
      "oop",
      "metaprogramming",
```
@Glasses-Neo, maybe remove "nim" and "nimble" tags?

---

That's right, thanks!